daemons/controld/controld_cib.c

```diff
@@ -832,14 +834,13 @@ int
 controld_update_cib(const char *section, xmlNode *data, int options,
                     void (*callback)(xmlNode *, int, int, xmlNode *, void *))
 {
+    cib_t *cib = controld_globals.cib_conn;
     int cib_rc = -ENOTCONN;
 
     CRM_ASSERT(data != NULL);
 
     if (controld_globals.cib_conn != NULL) {
```

Comment:
You can either use cib here or move cib into this block

Reply:
Good point